### PR TITLE
Modify BP policy to match with default validator

### PR DESCRIPTION
**Pull request by phuongnq**
https://github.com/craftersoftware/craftercms/issues/993

### CODE REVIEW

File: src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java [hunk 1]
```diff
@@ -207,7 +207,9 @@ public String writeContent(String siteId, String path, InputStream content) {
                             .replace(REPO_COMMIT_MESSAGE_USERNAME_VAR, username)
                             .replace(REPO_COMMIT_MESSAGE_PATH_VAR, path);
                     commitId = helper.commitFiles(repo, siteId, comment, user, path);
-                    insertProcessedCommitId(siteId, commitId);
+                    if (commitId != null) {
```
Comment (sumerjabri) [inline src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java]: This is not related to the ticket. Usually, we need a ticket associated with these updates.

Comment (jmendeza) [reply src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java]: @sumerjabri New ticket to take care of this part
https://github.com/craftersoftware/craftercms/issues/996

**Review by Euquimides — APPROVED** [review]
Tested the new `Project Policy Configuration` regex in different BPs. The regex is working fine against the validator.

File: src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java (hunk 1 above)
Comment (sumerjabri) [reply src/main/java/org/craftercms/studio/impl/v1/repository/git/GitContentRepository.java]: Added PR https://github.com/craftercms/studio/pull/3600

**Review by sumerjabri — APPROVED** [review]
